src/luagit2/config/lua_config_help.c

```diff
@@ -0,0 +1,100 @@
+#include "lua_config_help.h"
+
+// typedef enum {
+// 	/** System-wide on Windows, for compatibility with portable git */
+// 	GIT_CONFIG_LEVEL_PROGRAMDATA = 1,
+
+// 	/** System-wide configuration file; /etc/gitconfig on Linux systems */
+// 	GIT_CONFIG_LEVEL_SYSTEM = 2,
+
+// 	/** XDG compatible configuration file; typically ~/.config/git/config */
+// 	GIT_CONFIG_LEVEL_XDG = 3,
+
+// 	/** User-specific configuration file (also called Global configuration
+// 	 * file); typically ~/.gitconfig
+// 	 */
+// 	GIT_CONFIG_LEVEL_GLOBAL = 4,
+
+// 	/** Repository specific configuration file; $WORK_DIR/.git/config on
+// 	 * non-bare repos
+// 	 */
+// 	GIT_CONFIG_LEVEL_LOCAL = 5,
+
+// 	/** Application specific configuration file; freely defined by applications
+// 	 */
+// 	GIT_CONFIG_LEVEL_APP = 6,
+
+// 	/** Represents the highest level available config file (i.e. the most
+// 	 * specific config file available that actually is loaded)
+// 	 */
+// 	GIT_CONFIG_HIGHEST_LEVEL = -1,
+// } git_config_level_t;
```

Comment:
Why was this commented out?

Reply:
That is my mistake, I used to refer it while creating bindings. It has absolutely nothing to do with any of code. I will remove it. Thankyou